src/plone/testing/zope.py

```diff
@@ -951,8 +951,9 @@ def make_wsgi_app(self):
         Zope2.Startup.run.make_wsgi_app(global_config, zope_conf)
         app = ZPublisher.WSGIPublisher.publish_module
 
-        for spec, protocol, name, extra in reversed(self.pipeline):
-            entrypoint = pkg_resources.get_entry_info(spec, protocol, name)
+        all_entry_points = entry_points()
+        for _, protocol, name, extra in reversed(self.pipeline):
+            (entrypoint,) = all_entry_points.select(group=protocol, name=name)
```

Comment:
@gforcada I don't think this is compatible with Python 3.9. https://docs.python.org/3/library/importlib.metadata.html#entry-points says: "The “selectable” entry points were introduced in importlib_metadata 3.6 and Python 3.10. Prior to those changes, entry_points accepted no parameters and always returned a dictionary of entry points, keyed by group."

Reply:
@davisagli oh, right, thanks for noticing!

I updated this repo with the test matrix (see #108) and after merging and rebasing this one on top of it, indeed, all GHA pass but Python 3.9 on Plone 6.0